README.md

```diff
@@ -19,6 +19,7 @@ testing code that uses [jsverify](https://jsverify.github.io/) in
 
 What is a good invariant for your recursive implementation of `fib()`
 i.e. something that is always true at the beginning of the recursive call?
+----The previous two elements' fibonocci values.
```

Comment:
What about those values?

Reply:
In order for the fibonacci sequence to be correct up to the number we call, the prior two elements in the array must be correct. So the index of x-1 and x must be correct in order to find the next values since those are both variables to the next elements sum.

Reply:
Can you adjust your invariant accordingly please?